### DISCUSSION TITLE
AdGuard блочит

### DISCUSSION THREAD

**Discussion by voltdv** [bol-van/zapret#870] [Q&A]
при открытом AdGuard'е ломается дс. мб кто сталкивался/у кого то уже есть решение?

**Comment by HentayHentay** [comment]
У меня гениальный вопрос
А при чем тут запрет ?
На чем ком где....

**Comment by bol-van** [comment]
adguard использует kernel driver для перехвата трафика.
должно быть winws его дурит, а затем он перехватывается adguard, и он ломает дурение

**Comment by voltdv** [comment]
> adguard использует kernel driver для перехвата трафика. должно быть winws его дурит, а затем он перехватывается adguard, и он ломает дурение

это то понятно с других тредов. А фиксить как? Не то что бы нужно, не особо он и нужен как приложуха, но интереса ради

**Comment by HentayHentay** [comment]
приложение ради приложения... Гениально 
Фиксить - все же описано и ты сам уже написал..

Голова не только жевать

**Comment by Elllkere** [comment]
> А фиксить как?

TDI драйвер, если бы он был, ну или еслиб запрет можно было как прокси сделать, схожая ситуация как с VPNGame, но там есть TDI драйвер. Пиши в сап или просто используй днс сервера их

**Comment by 9iky6** [comment]
Уберите в настройках фильтрации трафика приложение дискорда из адварда. А вообще уже писали ранее это и в гугле есть.
